### DISCUSSION TITLE
Feature Request: List of radio buttons extension

### DISCUSSION THREAD

**Discussion by Better-IRIS** [ueberdosis/tiptap#4236] [Feature Requests]
### Description

I wish there was an extension to generate a list of radio buttons, similar to the list of checkboxes in the tasklist/taskitem extension.

### Use Case

I want to use this feature to quickly switch the value of the radio button.

### Type

New extension

**Comment by Kreachers** [comment]
This would be super cool to have. Does anybody have any idea how to add in radio buttons to a tiptap input field? Or any idea how to reuse the tasklist extension as a radio button group?
